packages/boilerplate/src/components/App.js

```diff
@@ -3,14 +3,46 @@ import { hot } from 'react-hot-loader'
 
 import Sidebar from './Sidebar'
 import Switcher from './Switcher'
+import { ConnectedRouter, Route } from '../router'
 
 import styles from '../css/App'
 
-const App = () => (
-  <div className={styles.app}>
-    <Sidebar />
-    <Switcher />
-  </div>
-)
+class App extends React.Component {
+  componentDidMount() {
+    console.log()
+  }
+
+  render() {
+    return (
+      <div className={styles.app}>
+        <Sidebar />
+        <Switcher />
+        <ConnectedRouter>
+          <Route
+            path="/test"
+            component={() => import('./Test')}
```

Comment:
This looks incompatible with `react-router`'s `component` prop (which is something that can be rendered with `React.createElement`). Perhaps The API should be left the same, and it be up to the user to use `react-universal-component` to achieve code splitting of the route if desired?